src/main/java/in/pune/royforge/eledgerapi/data/repo/ITransactionLogRepository.java

```diff
@@ -1,9 +1,15 @@
 package in.pune.royforge.eledgerapi.data.repo;
 
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.PagingAndSortingRepository;
 
 import in.pune.royforge.eledgerapi.data.entity.TransactionEntity;
 
-public interface ITransactionLogRepository extends PagingAndSortingRepository<TransactionEntity, String> {
+public interface ITransactionLogRepository extends PagingAndSortingRepository<TransactionEntity, Long> {
+
+	@Query("FROM TransactionEntity WHERE lenderId = ?1 AND borrowerId = ?2")
+	Iterable<TransactionEntity> transactionsList(String lenderId, String borrowerId);
```

Comment:
change return type of transactionsList method-->

List<TransactionEntity> transactionsList(String lenderId, String borrowerId);